tests/modules/test_convertobjectstoimage.py

```diff
@@ -1,180 +1,322 @@
+
+    assert len(pipeline.modules()) == 3
+
+    module = pipeline.modules()[2]
+
+    assert isinstance(module, cellprofiler.modules.convertobjectstoimage.ConvertToImage)
+
+    assert module.object_name.value == "Nuclei"
+
+    assert module.image_name.value == "NucleiImage"
+
+    assert module.image_mode.value == "Color"
+
+    assert module.colormap.value == "flag"
+
+
+def test_load_v1(pipeline):
+    data = ('eJztWltv2jAUNpeidpW6rnvYpL74cZvaKKFF2ngZFNaNqVy0ok57WxoM9WR'
+            'slDgU9kv2U/a4n7SfsBhCSbzQcGm5SIlkwXH8ne+c4xM7OXI5X7/In8GMos'
+            'Jyvn7cxATBGtF5k5ntLKT8CBZMpHPUgIxm4bmJ4WebQi0NtZOspmZPMzCtq'
+            'u/AfFesVN5zfn69BiDl/G47Le7e2nLlmKcJ+RJxjmnL2gJJ8NLt/+O0K93E'
+            '+jVBVzqxkTWmGPWXaJPV+527W2XWsAmq6G3vYOeq2O1rZFrV5gjo3q7hHiK'
+            'X+CeSXBgN+4K62MKMunhXv9x7x8u4xCviAJ6P4xCT4pBw2qGnX4z/BMbjkw'
+            'Fxe+YZv+/KmDZwFzdsnUDc1lt3Vgh9b0P0bUv6hFw1cevMCfk0+JSETw3ib'
+            'RCEh/y5EPy+hBetjnr8+ENPNzhs69y4WYYdOxJeyAVESEkE1BNPNURPzKcn'
+            'Bk6mnIctiV/Imnp0qi6ALzDCTDCd/08kvJCLDFLGoW2h6f1P+PQkwDcnG+e'
+            'dv1tMOTKn44378HFQYQ+HS0m40TXC7YBxfMKe3xeSn0IuoqZuEw4HuQaL2E'
+            'QGZ2Z/oXjPa/+s+a2A6fJrV/JbyFVu2fAjYdc6CeR/SLvnzQ/ZX21N7ZTzQ'
+            'D3SHtXOpI8vCYr5WmkenKqo2iJ2Lrq/5ULwQftCyVmWqIV5f0K8lhHnTbO7'
+            'wiiaJ/81dT3tlNeFTADfOtj5mPvDLLhciJ17wJ+vQh6+v1VtTjAVL7WrsHt'
+            'T4hvZuVo7nX1sLZ+rZb53RXmwvnbK+6qSWc2850LsDMrX+i2DBtEty61srM'
+            'LusO+6oLrMV4RbN6LM1hUFJWpMqiOsQ9yDvv/PmYlaJrNpY3H+7wez1cGW6'
+            'eegaCYc7UyvJyhP2fUP58t9rGjZ+ebhh5g2UGeGuATV4cZxGapb9boW4SJc'
+            'hItwm4zLeXDROhzhItxqcWHvWQfA/zwKmQ0rUv+9aG2S3xEuwq3Dfjft99i'
+            'm+BvhIlyEWx2uFxvj5DqTXK8d1KU8PEHr0xvgX5+EbCBCOiYT5+pMpT04/G'
+            'UphOmN4ekr5cL5W/IcxBI8nRCenMSTm8SDG4hy3Ox3TIfN5qytc2woJbe35'
+            'vTmR72C9yaENy3xpifxGox2kck5GzipFIZinXkOSMnzthPA541/3JGeHibv'
+            'nW8A/PM8nv+/7+fhiydiAz7vuYHdEFzSY9PIz99gtjx7dc/4kY/LGv8PmsR'
+            'r2g==')
+
+    fd = StringIO.StringIO(zlib.decompress(base64.b64decode(data)))
+
+    pipeline.load(fd)
+
+    assert len(pipeline.modules()) == 3
+
+    module = pipeline.modules()[2]
+
+    assert isinstance(module, cellprofiler.modules.convertobjectstoimage.ConvertToImage)
+
+    assert module.object_name.value == "Nuclei"
+
+    assert module.image_name.value == "CellImage"
+
+    assert module.image_mode.value == "Color"
+
+    assert module.colormap.value == "winter"
+
+
+def test_run_binary(workspace, module):
+    module.object_name.value = "inputobjects"
+
+    module.image_name.value = "outputimage"
+
+    module.image_mode.value = "Binary (black & white)"
+
+    module.run(workspace)
+
+    image = workspace.image_set.get_image("outputimage")
+
+    objects = workspace.get_objects("inputobjects")
+
+    assert image.dimensions == objects.segmented.ndim
+
+    pixel_data = image.pixel_data
+
+    assert pixel_data.shape == objects.shape
+
+    if objects.segmented.ndim is 2:
```

Comment:
Casing, like this, in tests makes me uncomfortable. We could potentially skip testing, for example, volumes while making it look like we are. Or, in the future, someone could make a change that would skip test from being run on some subset of inputs. 

I am wondering if we can avoid this casing in the tests by testing images and volumes separately. Test inputs could be 2D `segmented` and `ijv`, and 3D `segmented` (update with `ijv` when supported).